presto-main/src/main/java/io/prestosql/operator/scalar/JoniRegexpFunctions.java

```diff
@@ -307,4 +308,101 @@ private static void validateGroup(long group, Region region)
             throw new PrestoException(INVALID_FUNCTION_ARGUMENT, format("Pattern has %d groups. Cannot access group %d", region.numRegs - 1, group));
         }
     }
+
+    @ScalarFunction
+    @Description("returns the beginning position of the matched substring")
```

Comment:
`returns the index of the matched substring`. Change other descriptions similarly